src/components/TopBar/GridLogo.jsx

```diff
@@ -23,6 +23,9 @@ const styles = {
     clickable: {
         cursor: 'pointer',
     },
+    hideOnSmallScreen: {
+        display: { xs: 'none', lg: 'block' },
+    },
```

Comment:
place it in `title: {...}` style, the `style` property is passed to the `sx` inside `LogoText`

Reply:
Thanks, that's better.